### PR TITLE
west.yml: check nxp spi interrupt issue

### LINKED ISSUE #25566: LSPI of NXP i.MX RT Other interrupts treated as transfer completion
**Describe the bug**
After the master transfer is completed,if the sr flag is not clear, the callback of the transfer
completion will be called again in other interrupts.

**To Reproduce**
Call spi_wirte to send 150K Byte data , Measure the time used by spi_write, under normal circumstances it will take some time about  tens of milliseconds, this time will become 0 once issue occur. The occurrence of issues is random and cannot be recovered once it occur. The speed of SPI is set between 20M ~ 25M, which is relatively easy to copy.

**Additional context**
clean completion flag in isr can solve this problem, But need to discuss to decide whether to put this modification in HAL NXP or zephyr driver

### PR DESCRIPTION
**Pull request by lgl88911**
For CI check nxp spi interrupt issue update

Signed-off-by: Frank Li <lgl88911@163.com>

Fixes #25566

### CODE REVIEW

**Review by MaureenHelm — CHANGES_REQUESTED** [review]
Will you please change the commit message to something like:
```
west.yml: Update nxp hal to fix spi interrupt issue

Updates the nxp hal to incorporate a fix for other interrupts treated as a transfer completion.
```

File: west.yml [hunk 1]
```diff
@@ -95,7 +95,7 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 80a337dc4cc980f80e595971147eb41458b29ae8
+      revision: pull/44/head
```
Comment (MaureenHelm) [inline west.yml:98]: zephyrproject-rtos/hal_nxp#44 is now merged, so please update this line to the new SHA

**Review by MaureenHelm — CHANGES_REQUESTED** [review]
Thanks for updating the commit message

File: west.yml (hunk 1 above)
Comment (MaureenHelm) [reply west.yml]: Thanks for the update. Please also rebase to fix the merge conflict

Comment (lgl88911) [reply west.yml]: Thank you for your reminder, it's done

**Review by MaureenHelm — APPROVED** [review]

**Review by dleach02 — APPROVED** [review]

**Pull request merged**